.github/workflows/ci.yml

```
@@ -180,34 +180,33 @@ jobs:
             
-            # Analyze commit messages to determine version bump type
-            RECENT_COMMITS=$(git log --oneline -10 --pretty=format:"%s")
+            # Check for explicit version in PR title first
             PR_TITLE="${{ github.event.head_commit.message }}"
-            
-            echo "Recent commits:"
-            echo "$RECENT_COMMITS"
-            echo ""
             echo "PR Title: $PR_TITLE"
-            echo ""
             
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)